build.sh

```diff
@@ -5,14 +5,15 @@ cd "`dirname \"$0\"`"
 export REPOROOT=$(pwd)
 export GOPATH=$REPOROOT/.build
 export PATH=$GOPATH/bin:$PATH
+export CGO_CFLAGS_ALLOW='-fmodules|-fblocks'
```

Comment:
This is a fix to gomobile compliation issue --> https://github.com/golang/go/issues/23742